modules/fakemeta/fakemeta_amxx.cpp

```diff
@@ -13,12 +13,16 @@
 
 #include "fakemeta_amxx.h"
 #include "sh_stack.h"
+#include <reapi/mod_regamedll_api.h>
```

Comment:
Maybe will be better name it 'resdk/mod_regamedll_api.h' to avoid confusion with https://github.com/s1lentq/reapi?